experimental/Schemes/AffineSchemes.jl

```diff
@@ -110,7 +112,7 @@ base_ring_elem_type(X::AbsSpec) = base_ring_elem_type(typeof(X))
 On an affine scheme ``X/𝕜`` over ``𝕜`` this returns the ring `𝕜`.
 """
 function base_ring(X::AbsSpec{BRT, RT}) where {BRT, RT}
-  return base_ring(underlying_scheme(X))::BRT
+ return base_ring(underlying_scheme(X))::BRT
```

Comment:
It's not a clear rule, but we generally use two spaces for indentation; some code also uses 4; much rarer is 3; but 1 is IMO really "not enough" ;-)
```suggestion
  return base_ring(underlying_scheme(X))::BRT
```